java/osgi/osgi.bzl

```diff
@@ -174,7 +174,7 @@ def _osgi_jar_impl(ctx):
         source_jars = source_jars.to_list()
     if len(source_jars) > 1:
         fail("osgi_jar rule doesn't know how to deal with more than one source jar.")
-    source_jar = target_java_output.source_jars[0]
+    source_jar = target_java_output.source_jars.to_list()[0]
```

Comment:
You may need to rebase -- I believe this one was already fixed in https://github.com/protocolbuffers/protobuf/commit/925de189ecb8aa9b7dcd733ad46c651c12ca42d2

Reply:
Thank you, will do.